apps/crn-server/src/app.ts

```diff
@@ -589,16 +589,16 @@ export const appFactory = (libs: Libs = {}): Express => {
     interestGroupController,
     eventController,
   );
-  const labsRoutes = labsRouteFactory(labsController);
+  const labsRoutes = labRouteFactory(labsController);
```

Comment:
Should we be singularize the variable name as well?  If yes then apply to other route variables
```suggestion
  const labRoutes = labRouteFactory(labsController);
```